README.md

```diff
@@ -1,48 +1,78 @@
-## macOS install: 
+![u8timeseries](images/logo-72dpi.png "u8timeseries")
 
-The install happens with `pip`. For `conda` users, do the following first:
+**u8timeseries** is a python library for easily manipulating and forecasting time series in python.
+It contains a variety of models, from ARIMA to neural networks.
+The models can all be used in the same way, using `fit()` and `predict()` functions,
+similar to scikit-learn. The library also makes it easy to backtest models,
+and combine the predictions of several models and external regressors.
+
+### Install
+Run the following in your favorite python environment:
 ```
-conda install gcc
-conda install -c conda-forge fbprophet
+pip install u8timeseries
 ```
 
-Next, from the root of u8timeseries:
+##### Running the examples only without installing:
+To run the example notebooks without installing, using Docker, you can also run: 
 ```
-pip install .
+./build_docker.sh && ./run_docker.sh
 ```
+Then copy and paste the URL provided by the docker container into your browser to access Jupyter notebook.
 
-If Fortran is not installed on the device, the following error message might be printed during the installation:
+### Example Usage
```

Comment:
Example usage ?

Reply:
Only following the Chicago style of title capitalization here: https://capitalizemytitle.com/